docs/articles/index.md

```diff
@@ -9,9 +9,19 @@ This module provides the next generation Maintenance UI based on React/Typescrip
 As a prerequisite, you need:
 
 - NodeJS (9.5.0)
+- Python (2.7.18)
+- .Net Framework SDK 2.0
 - Visual Studio Code (latest)
 - Visual Studio > 2015
 
+Setting up a proper build evironment (in case you haven't already):
+
+*Step1.* Start PowerShell console as Admin
+
+*Step2.* `npm install -g windows-build-tools`
```

Comment:
Same here, this matches only for windows and the used shell should be independent. So if the user uses cmd, powershell or bash shouldn't matter.